client/apps/stripe-connect-account/view-wrapper.js

```diff
@@ -47,13 +49,23 @@ class StripeConnectAccountWrapper extends Component {
 	render() {
 		const {
 			isLoading,
+			isOAuthInitializing,
 			stripeConnectAccount,
+			stripeError,
 			isDeauthorizing,
 			isReloading,
 			oauthURL,
 			translate,
 		} = this.props;
 
+		if ( ! isOAuthInitializing && stripeError ) {
+			return (
+				<div className="stripe-connect-account__placeholder-container">
```

Comment:
Can this `<div>` wrapper be omitted? (Though the same goes for the `. stripe-connect-account__connect-action` below.)